### ISSUE TITLE
Update to RSQlite causes warnings

### ISSUE THREAD

**Issue by eliotmcintire** [pbiecek/archivist#316]
In `saveToLocalRepo`, `createLocalRepo` and likely others: many new warnings appearing, which appear to have come with `RSQLite 2.0` which is now on CRAN.

```
a <- 1
repo <- tempdir()
archivist::createLocalRepo(repo)
archivist::saveToLocalRepo(a, repoDir = repo)

Warning messages:
1: In rsqlite_fetch(res@ptr, n = n) :
  Don't need to call dbFetch() for statements, only for queries
```

**Comment by pbiecek** [comment]
thanks,
will check this along with #313

**Comment by pbiecek** [comment]
fixed with 3a0dff49f2cc78189c9077de9a90f5839ea66bd7

**Comment by MarcinKosinski** [comment]
Thanks for the report and fast fix : )
